### PR TITLE
Update CODEOWNERS for Expo packages

### PR DESCRIPTION
**Pull request by tsapeta**
Updated CODEOWNERS file so package owners will be automatically added as reviewers.

### CODE REVIEW

**Review by bbarthec — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,78 @@
...
+/packages/expo-brightness                 @sjchmiela
+/packages/expo-calendar                   @tsapeta
+/packages/expo-camera                     @bbarthec
+/packages/expo-camera-interface           @bbarthec
+/packages/expo-constants                  @tsapeta
+/packages/expo-constants-interface        @tsapeta
+/packages/expo-contacts                   @EvanBacon
+/packages/expo-core                       @sjchmiela
+/packages/expo-document-picker            @bbarthec
+/packages/expo-errors                     @tsapeta
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-errors                     @esamelson
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,2 +1,78 @@
...
+/packages/expo-firebase-functions         @EvanBacon
+/packages/expo-firebase-instance-id       @EvanBacon
+/packages/expo-firebase-invites           @EvanBacon
+/packages/expo-firebase-links             @EvanBacon
+/packages/expo-firebase-messaging         @EvanBacon
+/packages/expo-firebase-notifications     @EvanBacon
+/packages/expo-firebase-performance       @EvanBacon
+/packages/expo-firebase-remote-config     @EvanBacon
+/packages/expo-firebase-storage           @EvanBacon
+/packages/expo-font                       @tsapeta
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-font                       @sjchmiela
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,2 +1,78 @@
-/packages/babel-preset-expo     @brentvatne
-/packages/jest-expo             @brentvatne
+/packages/babel-preset-expo               @brentvatne
+/packages/expo-ads-admob                  @sjchmiela
+/packages/expo-ads-facebook               @sjchmiela
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-ads-facebook               @Szymon20000
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,2 +1,78 @@
...
+/packages/expo-firebase-instance-id       @EvanBacon
+/packages/expo-firebase-invites           @EvanBacon
+/packages/expo-firebase-links             @EvanBacon
+/packages/expo-firebase-messaging         @EvanBacon
+/packages/expo-firebase-notifications     @EvanBacon
+/packages/expo-firebase-performance       @EvanBacon
+/packages/expo-firebase-remote-config     @EvanBacon
+/packages/expo-firebase-storage           @EvanBacon
+/packages/expo-font                       @tsapeta
+/packages/expo-font-interface             @tsapeta
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-font-interface             @sjchmiela
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,2 +1,78 @@
...
+/packages/expo-sensors                    @sjchmiela
+/packages/expo-sensors-interface          @sjchmiela
+/packages/expo-sms                        @bbarthec
+/packages/expo-speech                     @Szymon20000
+/packages/expo-sqlite                     @Szymon20000
+/packages/expo-task-manager               @tsapeta
+/packages/expo-task-manager-interface     @tsapeta
+/packages/expo-web-browser                @sjchmiela
+/packages/expo-yarn-workspaces            @ide
+/packages/jest-expo                       @brentvatne
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/jest-expo                       @brentvatne
/packages/jest-expo/src/preset/expoModules.js
```

Would this work?

**Pull request merged**